hkmc2/shared/src/main/scala/hkmc2/semantics/Term.scala

```diff
@@ -897,6 +902,7 @@ object ClassDef:
       kind: ClsLikeKind,
       sym: ClassSymbol,
       bsym: BlockMemberSymbol,
+      ctorSym: Opt[TermSymbol],
```

Comment:
It seems we should make this
```suggestion
      ctorSym: S[TermSymbol],
```
right?

Parameterized classes should always have a ctor symbol.

Reply:
True